<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7SBKKJ1" failures="3" tests="11" name="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" time="99.020" errors="0" timestamp="2024-12-03T18:12:18 IST" skipped="0">
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Username" time="1.288"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Password" time="1.704"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Captcha" time="2.360"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="SUBMIT_Button" time="2.302"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Click_On_SMR" time="18.146"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Click_Reports" time="1.101"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Click_On_EW_CCP_bot_Lead_CONVERSION_REPORT" time="4.112"/>
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Dealer_Parent_Group_Wise_Tab" time="22.222">
    <failure message="No Data Found for Dealer Parent Group Wise In EW-CCP-lead conversion_Report : 0" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No Data Found for Dealer Parent Group Wise In EW-CCP-lead conversion_Report : 0
at org.testng.Assert.fail(Assert.java:99)
at SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports.Dealer_Parent_Group_Wise_Tab(CCM_Ew_CCp_Bot_Lead_Conversion_Reports.java:127)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Dealer_Parent_Group_Wise_Tab -->
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Workshop_Wise_Tab" time="22.898">
    <failure message="No Data Found for Workshop_Wise Wise In EW-CCP-lead conversion_Report : 0" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No Data Found for Workshop_Wise Wise In EW-CCP-lead conversion_Report : 0
at org.testng.Assert.fail(Assert.java:99)
at SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports.Workshop_Wise_Tab(CCM_Ew_CCp_Bot_Lead_Conversion_Reports.java:178)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Workshop_Wise_Tab -->
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Day_Wise_Tab" time="22.887">
    <failure message="No Data Found for Day_Wise Wise In EW-CCP-lead conversion_Report : 0" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: No Data Found for Day_Wise Wise In EW-CCP-lead conversion_Report : 0
at org.testng.Assert.fail(Assert.java:99)
at SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports.Day_Wise_Tab(CCM_Ew_CCp_Bot_Lead_Conversion_Reports.java:228)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Day_Wise_Tab -->
  <system-out/>
  <testcase classname="SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports" name="Comparison_0f_all_Data" time="0.000"/>
  <system-out/>
</testsuite> <!-- SMR_Call_Dispostion_Data_Check_And_CCM_Reports.CCM_Ew_CCp_Bot_Lead_Conversion_Reports -->
